guides/common/modules/proc_exporting-a-repository.adoc

```diff
@@ -34,14 +34,14 @@ For more information, see xref:Download_Policies_Overview_{context}[].
 * Ensure that you synchronize products that you export to the required date.
 
 .Procedure
-. Export a repository using the product name and repository name:
+. Export a repository using the product, repository, and organization names:
```

Comment:
This is a valid change that ensures all options passed with the command below are listed. Alternatively, to prevent issues like this in the future, you could also consider something like this:

```suggestion
. Export a repository:
```